### ISSUE TITLE
Child Class does not inherit Parent's class methods

### ISSUE THREAD

**Issue by brunzino** [hzamani/active_record-acts_as#29]
The Child class *does* inherit the instance methods, but not the class methods. Is this expected behavior? 

Here's a short code example
```
class Report < ActiveRecord::Base
  actable

  def self.current(user)
    where(:status => "open").accessible_by(user)
  end
end

class NightSupervisorReport < ActiveRecord::Base
  acts_as :report
end 
```

In the console
```
$ Report.current(User.first)
 => #<ActiveRecord::Relation []> 
$ NightSupervisorReport.current(User.first)
NoMethodError: undefined method `current' for #<Class:0x007f8fd4a206c8>
```

**Comment by hzamani** [comment]
It is the expected behavior.
What about this?
```ruby
class Report < ActiveRecord::Base
  actable

  scope :current, -> (user) { where(status: "open").accessible_by(user) }
  scope :night_supervisors, -> { where(actable_type: "NightSupervisorReport") }
end
```
and then
```
> Report.night_supervisors.current(User.first)
```

**Comment by manuelmeurer** [comment]
Fixed in v2.2.0
